File: solutions/python/hamming/1/hamming.py
def distance(strand_a, strand_b):
    d = 0
    if len(strand_a) != len(strand_b):
        raise ValueError("Value Error")
    for i in range(len(strand_a)):
        if strand_a[i] != strand_b[i]:
            d += 1
    return d

#print(distance("A", "GA"))

File: solutions/go/hamming/4/hamming.go
package hamming

import "errors"

func Distance(a, b string) (int, error) {
	

	if len(a) != len(b) {	
		return 0, errors.New("bad arguments")
	}
	var dist int;
	for i := range a {
		if a[i] != b[i] {
			dist++;
		}
	}
	return dist, nil

}
